network/p2p/libp2pUtils.go

```diff
@@ -249,3 +249,20 @@ func ExtractPeerID(networkPubKey fcrypto.PublicKey) (pid peer.ID, err error) {
 
 	return
 }
+
+// FlowPublicKeyFromPeerID converts a LibP2P peer ID to a Flow public key.
+func FlowPublicKeyFromPeerID(pid peer.ID) (networkPubKey fcrypto.PublicKey, err error) {
```

Comment:
This will not work for most nodes, and in fact will not work for our internal nodes.
https://github.com/onflow/flow-go/pull/1133/files#diff-f66caf91d2b6f3779515b2ef059291d0ed5de43bd384388bed50360a85df8192R18-R45
Are we sure we want this in libp2pUtils? If so, don't we want a big red comment on it?

Reply:
Yup, this was implemented before I realized it wouldn't work and I forgot to remove :)